doc/examples/segmentation/plot_watershed.py

```diff
@@ -54,7 +54,7 @@
 
 ax0.imshow(image, cmap=plt.cm.gray, interpolation='nearest')
 ax0.set_title('Overlapping objects')
-ax1.imshow(-distance, cmap=plt.cm.jet, interpolation='nearest')
+ax1.imshow(-distance, cmap=plt.cm.gray, interpolation='nearest')
```

Comment:
Nevermind, gray is probably one of the best options here.
I wanted to get terrain-like effect, but to get one the image with higher resolution and histogram equalization are required.